core/src/main/java/org/axonframework/eventhandling/EventProcessor.java

```diff
@@ -59,6 +61,13 @@
      */
     Registration registerInterceptor(MessageHandlerInterceptor<? super EventMessage<?>> interceptor);
 
+    /**
+     * Return the list of registered interceptors for this event processor
```

Comment:
I'd adjust the javadoc to talk about `{@link MessageHandlerInterceptor}s` i.o. `intereptors`.

Reply:
Super nit: And please add a dot at the end of this line :-)